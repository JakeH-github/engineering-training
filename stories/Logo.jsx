import React from 'react';

export default function Logo({ logo: { title, state } }) {
    return (
    <img src="https://www.totalwine.com/global-static-resources/@totalwinelabs/twc-icon/3.13.0/twm-logo-color.svg#twm-logo-color" aria-label={title} className="logo" alt="total wine logo"/>
    );
}

