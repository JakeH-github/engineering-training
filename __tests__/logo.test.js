import renderer from 'react-test-renderer';
import Logo from "../src/components/logo";
import React from "react";

test('Render the logo', () => {
    const component = renderer.create(
        <Logo/>,
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
