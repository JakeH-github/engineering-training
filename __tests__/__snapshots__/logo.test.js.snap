// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the logo 1`] = `
<img
  alt="total wine logo"
  className="logo"
  src="https://www.totalwine.com/global-static-resources/@totalwinelabs/twc-icon/3.13.0/twm-logo-color.svg#twm-logo-color"
/>
`;
